ifsbench/data/namelisthandler.py

```diff
@@ -164,7 +166,86 @@ def execute(self, wdir, **kwargs):
         debug(f"Modify namelist {input_path}.")
         namelist = f90nml.read(input_path)
 
+        namelist.uppercase = True
+        namelist.end_comma = True
+
         for override in self.overrides:
             override.apply(namelist)
 
         namelist.write(output_path, force=True)
+
+class SanitiseMode(Enum):
+    """
+    Specify, how duplicated entries in a namelist should be sanitised.
+    """
+
+    #: For multiply defined namelist groups, retain only the first.
+    FIRST = 'first'
```

Comment:
Please update namelist.py to use the enums rather than the hard-coded strings. May need moving this Enum to avoid a circular dependency.